scripts/reporting/batch_resolve_redcap_update_summary_scores.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+##
+##  See COPYING file distributed along with the ncanda-data-integration package
+##  for the copyright and license terms
+##
+
+import argparse
+import pandas as pd
+from pathlib import Path
+import sibispy
+import sibispy.cli
+from sibispy import sibislogger as slog
+import sys
+from typing import Sequence, List, Dict, Mapping, Union
+import yaml
+
+import github
+import pdb
+
+def run_batch():
+    script_path = Path("/sibis-software/python-packages/sibispy/cmds/")
+    events = ["Baseline", "1y", "2y", "3y", "4y", "5y", "6y"]
+    updates_scores_base_command = [script_path / "redcap_update_summary_scores", "-a", "-s"]
+    locking_data_base_command = [script_path / "exec_redcap_locking_data", "-e"] + events + ["-f", "Clinical", "-s"]
```

Comment:
`clinical` might be case-sensitive, and you should check if the form to be unlocked doesn't need to be `highrisk` or `brief`